modules/ols-large-language-model-requirements.adoc

```diff
@@ -3,16 +3,21 @@
 = Large Language Model (LLM) requirements 
 :context: ols-large-language-model-requirements
 
-As part of the {ols-release}  release, {ols-long} relies on Software as a Service (SaaS) LLM providers. You will need to have either a trial or paid subscription that allows for API access to completions and inferences with one of the following providers:
+As part of the {ols-release} release, {ols-long} often works with Software as a Service (SaaS) Large Language Model (LLM) providers. For these installations, you need to have either a trial or paid subscription that allows for API access to completions and inferences with one of the following providers:
 
 * OpenAI
 
-* Azure OpenAI
+* {azure-openai}
 
-* WatsonX
+* {watsonx}
+
+* {rhelai}
+
+* {rhoai}
 
 [NOTE]
 ====
 Many self-hosted or self-managed model servers claim API compatibility with OpenAI. It is possible to configure the OpenShift Lightspeed OpenAI provider to point to an API-compatible model server. If the model server is truly API-compatible, especially with respect to authentication, then it may work. These configurations have not been tested by Red Hat, and issues related to their use are outside the scope of {ols-release} support.
 ====
 
+For {ols-long} installations with {rhelai} or {rhoai}, you host your own model providers.
```

Comment:
As part of the {ols-release} release, {ols-long} can rely on Software as a Service (SaaS) LLM providers:

* OpenAI
* Azure OpenAI
* WatsonX

For API access to completions and inferences with one of these, you will need to have either a trial or paid subscription.

For {ols-long} installations with {rhelai} or {rhoai}, you host your own model providers.

Reply:
done